tests/shared/src/test/scala/org/http4s/headers/XFrameOptionsSuite.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2013 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s
+package headers
+
+import org.http4s.implicits.http4sSelectSyntaxOne
+
+class XFrameOptionsSuite extends Http4sSuite {
```

Comment:
Oh actually, we should test laws like we do for other headers e.g.
https://github.com/http4s/http4s/blob/4c777033f2ab8662dc5c56cafb139438252b42fb/tests/shared/src/test/scala/org/http4s/headers/AcceptHeaderSuite.scala#L23

I think this was missing in your other PR https://github.com/http4s/http4s/pull/6981 as well.